pkg/apis/flyteworkflow/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -549,7 +550,7 @@ func (in *NodeStatus) DeepCopyInto(out *NodeStatus) {
 		*out = (*in).DeepCopy()
 	}
 	if in.DataReferenceConstructor != nil {
-		out.DataReferenceConstructor = in.DataReferenceConstructor
+		out.DataReferenceConstructor = in.DataReferenceConstructor.DeepCopyReferenceConstructor()
```

Comment:
this is not compiling right?
you might want to just revert this line, or add a DeepCopy that just returns self